protocol/src/protocol_generated.rs

```diff
@@ -4115,4 +4115,3 @@ pub fn finish_size_prefixed_sync_message_buffer<'a, 'b>(fbb: &'b mut flatbuffers
 }
 }  // pub mod Protocol
 }  // pub mod Ckb
-
```

Comment:
this file is generated.

Reply:
I think it is too strick to check the blank new lines at the end of the file.

Reply:
Just disable the eof blank lines checker.